canonical-web-and-design/canonicalwebteam.blog
# Adds Atom feed View to both Django and Flask

Using werzeugs Atom Feed generator.

## QA

Implement the blog module in your project in `requirements.txt` by adding `git+https://github.com/b-m-f/blog-extension@add_atom_feed#egg=canonicalwebteam.blog`.

Consume the feed with `from canonicalwebteam.blog.django.views import feed`.

The parameters are `def feed(request, tags_exclude=[], tags=[], title="", subtitle=""):`.

Point a url to this view and confirm that a correct XML is returned.
